theatre/core/src/propTypes/index.ts

```diff
@@ -234,6 +242,100 @@ const _interpolateNumber = (
   return left + progression * (right - left)
 }
 
+export const rgba = (
+  defaultValue: Rgba,
+  opts?: {
+    label?: string
+    sanitize?: Sanitizer<Rgba>
+    interpolate?: Interpolator<Rgba>
```

Comment:
Also no need for a user-defined interpolator. They make sense on more general types like strings, whose value could have different formats and the user may want to interpolate between the components of those formats.